data/json/mapgen/triffid/triffid_fields.json

```diff
@@ -30,8 +30,7 @@
         "                    X   ",
         "       *          I     ",
         "     X     ^            ",
-        "            X           ",
-        "      I            I    "
+        "            X           "
```

Comment:
Any particular reason to delete the row at the bottom with two `I`s - there is one blank row above?

Reply:
No particular reason.  When in doubt I just deleted the last line.

Reply:
Seeing as there are no other blank rows in this map, I would assume that's the actual error.